v1/tests/test_order.py

```diff
@@ -20,7 +20,7 @@ def test_make_order(self):
         self.assertEqual(response.status_code, 201)
```

Comment:
It's always good to start with the expected value than starting with the actual value. Following this practice gives a clear message when other people run the tests.

Reply:
Am on it thanks.